after-install.sh

```diff
@@ -9,7 +10,9 @@ create_user() {
   FILES="${FILES} /usr/local/bin/lua2json"
   FILES="${FILES} /usr/local/bin/luarocks"
   FILES="${FILES} /usr/local/bin/luarocks-admin"
+  FILES="${FILES} /usr/local/bin/openapi2kong"
   FILES="${FILES} /usr/local/etc/luarocks/"
+  FILES="${FILES} /usr/local/etc/passwdqc/"
```

Comment:
these two addition are enterprise only are they not?

currently KBT build kong
kong-distributions builds kong enterprise

that changes in 3.0 (hopefully) with https://github.com/Kong/kong-build-tools/pull/455

Reply:
@hutchic , yep, currently it is. I added this two lines to make "kong-distributions" (EE) and "kong-build-tools" (CE) synced.

According to @gszr, we will deprecate "kong-distributions" in favor of "kong-build-tools". So, along this PR, I added the extra two lines.

A similar PR to is created for the enterprise version now: https://github.com/Kong/kong-distributions/pull/773

Reply:
In that case I recommend changing the:

```
chown -R kong:kong ${FILE}
chmod -R g=u ${FILE}
```

below to:

```
chown -f -R kong:kong ${FILE}
chmod -f -R g=u ${FILE}
```

so to avoid potential errors during OSS install.